### ISSUE TITLE
`imProc.astrometry.astrometryRefine(AI_array)` adds  RA and Dec only to the Catalog of the first image

### ISSUE THREAD

**Issue by EastEriq** [EranOfek/AstroPack#173]
```
>> [astrometry,AI]=imProc.astrometry.astrometryRefine(AI);

>> numel(AI)

ans =

     3

>> AI.CatData

ans = 

  AstroCatalog with properties:

           JD: []
      Catalog: [455×53 double]
     ColNames: {1×53 cell}
     ColUnits: {1×53 cell}
      ColDesc: {}
    SortByCol: 53
     IsSorted: 1
         Name: []
     DataType: Cat
       Config: [1×1 Configuration]
       Logger: [1×1 MsgLogger]
     UserData: []


ans = 

  AstroCatalog with properties:

           JD: []
      Catalog: [562×51 double]
     ColNames: {1×51 cell}
     ColUnits: {1×51 cell}
      ColDesc: {}
    SortByCol: []
     IsSorted: 0
         Name: []
     DataType: Cat
       Config: [1×1 Configuration]
       Logger: [1×1 MsgLogger]
     UserData: []


ans = 

  AstroCatalog with properties:

           JD: []
      Catalog: [534×51 double]
     ColNames: {1×51 cell}
     ColUnits: {1×51 cell}
      ColDesc: {}
    SortByCol: []
     IsSorted: 0
         Name: []
     DataType: Cat
       Config: [1×1 Configuration]
       Logger: [1×1 MsgLogger]
     UserData: []

>> AI(1).CatData.ColNames

ans =

  1×53 cell array

  Columns 1 through 7

    {'XPEAK'}    {'YPEAK'}    {'X1'}    {'Y1'}    {'X2'}    {'Y2'}    {'XY'}

  Columns 8 through 13

    {'SN_1'}    {'SN_2'}    {'SN_3'}    {'SN_4'}    {'SN_5'}    {'BACK_IM'}

  Columns 14 through 17

    {'VAR_IM'}    {'BACK_ANNULUS'}    {'STD_ANNULUS'}    {'FLUX_APER_1'}

  Columns 18 through 21

    {'FLUX_APER_2'}    {'FLUX_APER_3'}    {'FLUXERR_APER_1'}    {'FLUXERR_APER_2'}

  Columns 22 through 25

    {'FLUXERR_APER_3'}    {'MAG_APER_1'}    {'MAG_APER_2'}    {'MAG_APER_3'}

  Columns 26 through 29

    {'MAGERR_APER_1'}    {'MAGERR_APER_2'}    {'MAGERR_APER_3'}    {'FLUX_CONV_1'}

  Columns 30 through 33

    {'FLUX_CONV_2'}    {'FLUX_CONV_3'}    {'FLUX_CONV_4'}    {'FLUX_CONV_5'}

  Columns 34 through 37

    {'MAG_CONV_1'}    {'MAG_CONV_2'}    {'MAG_CONV_3'}    {'MAG_CONV_4'}

  Columns 38 through 41

    {'MAG_CONV_5'}    {'MAGERR_CONV_1'}    {'MAGERR_CONV_2'}    {'MAGERR_CONV_3'}

  Columns 42 through 46

    {'MAGERR_CONV_4'}    {'MAGERR_CONV_5'}    {'FLAGS'}    {'X'}    {'Y'}

  Columns 47 through 51

    {'FLUX_PSF'}    {'MAG_PSF'}    {'MAGERR_PSF'}    {'PSF_CHI2DOF'}    {'SN'}

  Columns 52 through 53

    {'RA'}    {'Dec'}

>> AI(3).CatData.ColNames

ans =

  1×51 cell array

  Columns 1 through 7

    {'XPEAK'}    {'YPEAK'}    {'X1'}    {'Y1'}    {'X2'}    {'Y2'}    {'XY'}

  Columns 8 through 13

    {'SN_1'}    {'SN_2'}    {'SN_3'}    {'SN_4'}    {'SN_5'}    {'BACK_IM'}

  Columns 14 through 17

    {'VAR_IM'}    {'BACK_ANNULUS'}    {'STD_ANNULUS'}    {'FLUX_APER_1'}

  Columns 18 through 21

    {'FLUX_APER_2'}    {'FLUX_APER_3'}    {'FLUXERR_APER_1'}    {'FLUXERR_APER_2'}

  Columns 22 through 25

    {'FLUXERR_APER_3'}    {'MAG_APER_1'}    {'MAG_APER_2'}    {'MAG_APER_3'}

  Columns 26 through 29

    {'MAGERR_APER_1'}    {'MAGERR_APER_2'}    {'MAGERR_APER_3'}    {'FLUX_CONV_1'}

  Columns 30 through 33

    {'FLUX_CONV_2'}    {'FLUX_CONV_3'}    {'FLUX_CONV_4'}    {'FLUX_CONV_5'}

  Columns 34 through 37

    {'MAG_CONV_1'}    {'MAG_CONV_2'}    {'MAG_CONV_3'}    {'MAG_CONV_4'}

  Columns 38 through 41

    {'MAG_CONV_5'}    {'MAGERR_CONV_1'}    {'MAGERR_CONV_2'}    {'MAGERR_CONV_3'}

  Columns 42 through 46

    {'MAGERR_CONV_4'}    {'MAGERR_CONV_5'}    {'FLAGS'}    {'X'}    {'Y'}

  Columns 47 through 51

    {'FLUX_PSF'}    {'MAG_PSF'}    {'MAGERR_PSF'}    {'PSF_CHI2DOF'}    {'SN'}

```

**Comment by EastEriq** [comment]
The reason is very likely the same as for issue #174. `.CatData` is updated only if `Nmatches<Args.MinNmatches` (lines 459 ff in astronomyRefine), which is not for the images beyond 1.

**Comment by EastEriq** [comment]
So, this should be solved now by commit 844b6302, in the process of dealing with issue #170

**Comment by EranOfek** [comment]
This is related to issue #174.
Fixed
